admin/query/src/main/java/org/codice/ditto/replication/admin/query/ReplicationUtils.java

```diff
@@ -107,7 +105,11 @@ public boolean deleteSite(String id) {
   }
 
   public boolean replicationConfigExists(String name) {
-    return configManager.objects().map(ReplicatorConfig::getName).anyMatch(name::equals);
+    return configManager.objects().map(ReplicatorConfig::getName).anyMatch(name::equalsIgnoreCase);
+  }
+
+  public boolean configExists(String id) {
+    return configManager.objects().map(ReplicatorConfig::getId).anyMatch(id::equals);
```

Comment:
❓ Wouldn't it be more optimal to use `configManager.get(id)` as opposed to load all configs?

✏️ These and the one above might be good candidate to move into the `ReplicatorConfigManager` class.